selftests/test_deproxy_message.py

```diff
@@ -188,7 +188,7 @@ def test_request_plain(self):
             "\r\n"
         )
         headers = ["Host: 10.0.10.2", "User-Agent: curl/7.53.1", "Accept: */*"]
-        created = deproxy.Request.create("GET", headers)
+        created = deproxy.Request.create("GET", headers, authority=False)
```

Comment:
When I see `authority=False` I expect that authority is boolean flag, but authority is string, this is confusing me little bit.